data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -114,6 +114,78 @@
       <arg type="o" name="handle" direction="out"/>
     </method>
 
+    <!--
+        CreateDraftSession:
+        @options: Vardict with optional further information
+        @session_handle: Object path for the created :ref:`org.freedesktop.portal.Session` object
+
+        Creates a draft capture input session. A successfully created session can at
+        any time be closed using :ref:`org.freedesktop.portal.Session.Close`, or may
+        at any time be closed by the portal implementation, which will be
+        signalled via :ref:`org.freedesktop.portal.Session::Closed`.
+
+        Supported keys in the @options vardict include:
+
+        * ``session_handle_token`` (``s``)
+
+          A string that will be used as the last element of the session handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Session` documentation for
+          more information about the session handle.
+
+        This method was added in version 2 of this interface.
+    -->
+    <method name="CreateDraftSession">
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In1" value="QVariantMap"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="session_handle" direction="out"/>
+    </method>
+
+    <!--
+        InitSession:
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+
+        Initializes a capture input session. This may result the portal backend querying the user whether
```

Comment:
```suggestion
        Initializes a capture input session previously created with CreateDraftSession. This may result the portal backend querying the user whether
```

Reply:
Changed the description to 
```
 This method may only be called once and only on a session previously created with CreateDraftSession.
 Initializes a capture input session previously created with CreateDraftSession. This may cause
 the portal backend to query the user whether they want to allow the input of the session to be
 captured or not, and what capabilities to support.
```